package exception;

public class Test2 {

	public static void main(String[] args) {
	throw new ArithmeticException();

	}

}
